src/style.css

```diff
@@ -1,3 +1,113 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  padding-top: 1px;
+
+  --header-height: 60px;
+  --font-size: 12px;
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  border: 0;
+
+  /* max-width: 1200px;
+  min-width: 1024px; */
+
+  width: 100%;
+  box-sizing: border-box;
+  align-items: center;
+  outline: none;
+}
+
+/* @media (min-width: 1024px) and (max-width: 1200px) {
+  .header {
+    padding: 0 50px;
+  }
+} */
+
+.header:hover {
+  outline: 2px solid var(--link-color);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  outline: none;
+}
+
+.logo:hover {
+  outline: 2px solid var(--link-color);
+}
+
+.nav {
+  display: flex;
+  height: 60px;
+
+  /* padding-top: 1px; */
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline-start: 0;
+}
+
+.nav__item {
+  list-style-type: none;
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). Instead, style elements using '.class'. In this case, you can just use the already defined '.nav__item' class to set the list-style-type to 'none'.